src/main/java/calculator/StringAddCalculator.java

```diff
@@ -0,0 +1,60 @@
+package calculator;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringAddCalculator {
+
+    private static final int ZERO = 0;
+    private static final String REGEX_DELIMITER = "|";
+    private static final List<String> STANDARD_DELIMISTERS = Arrays.asList(",", ":");
+    private static final String CUSTOM_REGEX = "//(.)\n(.*)";
+
+    private StringAddCalculator() {
+    }
+
+    public static int splitAndSum(final String expression) {
+        if (isNullOrEmpty(expression)) {
+            return ZERO;
+        }
+
+        List<String> stringNumbers = split(expression);
+
+        return sumPositiveNumbers(stringNumbers);
+    }
+
+    private static boolean isNullOrEmpty(final String expression) {
+        return Objects.isNull(expression) || expression.isEmpty();
+    }
+
+    private static List<String> split(final String expression) {
+        Matcher m = Pattern.compile(CUSTOM_REGEX).matcher(expression);
+        if (m.find()) {
+            String customDelimiter = m.group(1);
+            return Arrays.asList(m.group(2).split(customDelimiter));
+        }
+
+        String regex = makeStandardDelimiterRegex();
+        return Arrays.asList(expression.split(regex));
+    }
+
+    private static int sumPositiveNumbers(final List<String> stringNumbers) {
+        return stringNumbers.stream()
+                .mapToInt(Integer::parseInt)
+                .peek(StringAddCalculator::checkNegativeNumber)
+                .sum();
+    }
+    
+    private static void checkNegativeNumber(final int number) {
+        if (number < 0) {
```

Comment:
정의해 둔 상수를 사용할 수 있을 것 같아요 😄 
```suggestion
        if (number < ZERO) {
```

Reply:
놓쳤던 부분이네요. 코멘트 감사합니다🙏🏻